doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -444,10 +444,13 @@ not its name, one usually uses “arrow” abstractions for prenex
 arguments, or the ``(_ : term)`` syntax for inner arguments. In |SSR|,
 the latter can be replaced by the open syntax ``of term`` or
 (equivalently) ``& term``, which are both syntactically equivalent to a
-``(_ : term)`` expression.
+``(_ : term)`` expression:
+
+.. prodn::
+   binder += binder | & term | of term
```

Comment:
For the grammar tokens at the left of the `+=` in a `prodn` directive, you need to put an `@` in front. This would give `binder += @binder | & @term | of @term`.